tripal/src/TripalStorage/PluginManager/TripalStorageManager.php

```diff
@@ -17,25 +17,23 @@ class TripalStorageManager extends DefaultPluginManager {
    * @param \Traversable $namespaces
    *   An object that implements \Traversable which contains the root paths
    *   keyed by the corresponding namespace to look for plugin implementations.
+   * @param Drupal\Core\Cache\CacheBackendInterface $cache_backend
+   *   The backend to use when caching this plugin type.
    * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
    *   The module handler.
-   * @param string $plugin_interface
-   *   The interface each plugin should implement.
-   * @param string $plugin_definition_annotation_name
-   *   The name of the annotation that contains the plugin definition.
    */
   public function __construct(
-      \Traversable $namespaces
-      ,CacheBackendInterface $cache_backend
-      ,ModuleHandlerInterface $module_handler
+    \Traversable $namespaces,
+    CacheBackendInterface $cache_backend,
+    ModuleHandlerInterface $module_handler,
   ) {
     parent::__construct(
-        "Plugin/TripalStorage"
-        ,$namespaces
-        ,$module_handler
-        ,'Drupal\tripal\TripalStorage\Interfaces\TripalStorageInterface'
-        ,'Drupal\tripal\TripalStorage\Attribute\TripalStorage'
-        ,'Drupal\tripal\TripalStorage\Annotation\TripalStorage'
+        "Plugin/TripalStorage",
```

Comment:
```suggestion
        'Plugin/TripalStorage',
```
While we're in here, make quoting consistent with the other parameters.